components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -476,6 +491,7 @@
 <config_btr_dt ocn_grid="SOwISC12to60E2r4">'0000_00:00:15'</config_btr_dt>
 <config_btr_dt ocn_grid="ECwISC30to60E2r1">'0000_00:01:00'</config_btr_dt>
 <config_btr_dt ocn_grid="ECwISC30to60E3r2">'0000_00:01:00'</config_btr_dt>
+<config_btr_dt ocn_grid="IcoswISC30E3r2">'0000_00:00:45'</config_btr_dt>
```

Comment:
(documenting a slack conversation) This icos ~30km  should be able to run with exactly the same time step settings as the EC30to60, since it only depends on the minimum cell width. That means
```
<config_dt ocn_grid="IcoswISC30E3r2">'00:30:00'</config_dt>  (it's already set to that)
<config_btr_dt ocn_grid="IcoswISC30E3r2">'0000_00:01:15'</config_btr_dt>
```
The 45s was overly cautious to begin with. 1:15 should work.

Reply:
OK, thanks. The barotropic time step is limited by an advective CFL condition for the surface gravity waves. There must be some faster velocities on the ICOS30 in locations with 30km cells, where the EC30to60 has larger cells (like mid-latitude western boundary currents). Regardless, we always set this by trial and error in the end. Thanks for testing it. The 1:00 is still a big improvement over 45s.